packages/models-library/src/models_library/api_schemas_catalog/services.py

```diff
@@ -205,7 +205,10 @@ class ServiceUpdate(ServiceMetaDataEditable, ServiceAccessRights):
 class ServiceGet(
     ServiceMetaDataPublished, ServiceAccessRights, ServiceMetaDataEditable
 ):  # pylint: disable=too-many-ancestors
-    owner: LowerCaseEmailStr | None
+    owner: LowerCaseEmailStr | None = Field(
+        None,
```

Comment:
this change seems wrong. the ServiceGet comes from the catalog. this changes the API are you sure?

Reply:
I know we have the following scenario:
-> push service to osparc registry who's email is not part of the users
-> this owner filed is set to None

This should cover this case where. Since an integration test was failing this helped with fixing it. I'm not 100% sure if it might break anything though.

Reply:
this is the API of the catalog so I guess this should not be changed